modules/@ergonode/ui/src/components/ParagraphWithLinks/ParagraphWithLinks.vue

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<script>
+import {
+    isEmpty,
+} from '@Core/models/objectWrapper';
+
+export default {
+    name: 'ParagraphWithLinks',
+    props: {
+        paragraph: {
+            type: String,
+            required: true,
+        },
+        links: {
+            type: Object,
+            default: () => ({}),
```

Comment:
Passing **links** as an object is changing the way vue is communicating between components. This approach is perfect for React, not for Vue. Ofc there are cases that It would be pretty damn hard to avoid this pattern, but tho It's solution for slots and returning callbacks in them. Also by changing **span** to **a** tag it will solve problem, cuz **a** has `href` attribute which will match it. In case that we want do other action **a** tag might support `onclickevent` which will emit 'click' to the parent.